### PR TITLE
feat(collector): changed restart collection button design

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
- changed restart collection button design
AS-IS
image
TO-BE
image

- feat(collector): apply changed design at current status
AS-IS
image
TO-BE
image

- apply refresh API after restart collector

### Things to Talk About
🚨 plz check by commits

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorContentItem.vue [hunk 1]
```diff
@@ -94,8 +95,9 @@ const handleClickCollectData = async () => {
     } else {
         const collectorId = props.item.collectorId;
         await collectorPageStore.restartCollector(collectorId);
-        emit('refresh-collector-list');
     }
+    state.loading = false;
+    emit('refresh-collector-list');
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorContentItem.vue]: suggestion: How about putting it inside the `finally` block?

File: apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorContentItem.vue [hunk 2]
```diff
@@ -81,20 +83,19 @@ const state = reactive({
...
+        const collectorCollector = collectorPageStore.collectors.find((collector) => collector.collector_id === props.item.collectorId);
+        collectorPageStore.$patch({
+            visibleRestartModal: true,
+            selectedCollector: collectorCollector,
+        });
+    } else {
+        const collectorId = props.item.collectorId;
+        await collectorPageStore.restartCollector(collectorId);
+        emit('refresh-collector-list');
+    }
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorContentItem.vue]: quibble: This part doesn't seem to fit with this commit message!

**Review by yuda110 — APPROVED** [review]
LGTM!

File: apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorContentItem.vue [hunk 3]
```diff
@@ -27,7 +27,7 @@
             </div>
             <div class="collector-status-wrapper">
                 <p-button style-type="tertiary"
-                          :loading="collectorPageState.collectorLoading"
+                          :loading="state.collectorLoading"
```
Comment (sulmoJ) [inline apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorContentItem.vue:30]: issue: `CollectorLoading` does not seem to exist in the `state` object!

**Review by sulmoJ — COMMENTED** [review]
I left a comment!

**Review by sulmoJ — APPROVED** [review]
LGTM!

**Pull request merged**